src/test/java/com/conveyal/gtfs/TestUtils.java

```diff
@@ -156,4 +159,30 @@ private static void compressDirectoryToZipfile(String rootDir, String sourceDir,
     public static String fileNameWithDir(String directory, String filename) {
         return String.join(File.separator, directory, filename);
     }
+
+    /**
+     * Asserts that the result of a SQL count statement is equal to an expected value
+     *
+     * @param sql A SQL statement in the form of `SELECT count(*) FROM ...`
+     * @param expectedCount The expected count that is returned from the result of the SQL statement.
+     */
+    public static void assertThatSqlCountQueryYieldsExpectedCount(DataSource dataSource, String sql, int expectedCount) {
+        int count = -1;
+        LOG.info(sql);
+        // Encapsulate connection in try-with-resources to ensure it is closed and does not interfere with other tests.
+        try (Connection connection = dataSource.getConnection()) {
+            ResultSet resultSet = connection.prepareStatement(sql).executeQuery();
+            while (resultSet.next()) {
```

Comment:
If using a `select count` query, do we need the `while` loop to get the result?

```suggestion
            if (resultSet.next()) {
```

Reply:
Updated.